### ISSUE TITLE
[Segmented] Narrator doesn't announce ControlType / Selection state

### ISSUE THREAD

**Issue by niels9001** [CommunityToolkit/Windows#192]
From WinUI-Gallery repo:

**Pre-Requisite:**
Turn on narrator with shortcut key "win+ctrl+Enter" key.

**Repro-Steps:**
Launch the WinUI app. 
Navigate to search box using tab key.
Type “Colors” and hit enter.
Navigate to Colors card and activate it.
Navigate between the Top banner items i.e., Text, Fill, Stroke, Background and Signal
Listen to the narrator announcement.

**Actual Result:**
Control type is not defined to the Top banner items i.e., Text, Fill, Stroke, Background and Signal and when user invoke any of the item there is no selection state information announced.

When user navigates through the items narrator simply announces the label and when user invoke any of the item screen reader remains silent. Fails to announce the selection state of the button.

**Expected Result:**
Control type should be defined to the Top banner items i.e., Text, Fill, Stroke, Background and Signal and Narrator should announce the selection state of the buttons as Selected/Unselected.

**User Impact:**
User with visual impairments who rely on screen reader will face difficulties and will not be able to know the control type and state if narrator fails to announce.
